src/Sql/Sql.Test/ScenarioTests/SqlTestRunner.cs

```diff
@@ -0,0 +1,190 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Test.HttpRecorder;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using System.Collections.Generic;
+using Microsoft.Azure.Management.Internal.Resources;
+using Microsoft.Azure.Management.Sql;
+using CommonStorage = Microsoft.Azure.Management.Storage.Version2017_10_01;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit.Abstractions;
+using Microsoft.Azure.Management.EventHub;
+using Microsoft.Azure.Management.OperationalInsights;
+using SDKMonitor = Microsoft.Azure.Management.Monitor;
+using CommonMonitor = Microsoft.Azure.Management.Monitor.Version2018_09_01;
+using Microsoft.Azure.Management.KeyVault;
+using Microsoft.Azure.Graph.RBAC;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.TestFx;
+
+namespace Microsoft.Azure.Commands.ScenarioTest.SqlTests
+{
+    public class SqlTestRunner
+    {
+        protected string[] resourceTypesToIgnoreApiVersion = new string[] {
```

Comment:
This change is at risk. Originally it was trying to set different resource types to ignore for different test classes. With this variable, it will ignore all of the following types for all test cases. Please try to see if you have another way to resolve it. If not, please ping me.